src/winforms/toga_winforms/widgets/canvas.py

```diff
@@ -22,10 +22,20 @@ class WinformContext(Context):
     def __init__(self):
         super(WinformContext, self).__init__()
         self.graphics = None
-        self.path = None
+        self.paths = []
         self.start_point = None
         self.last_point = None
 
+    @property
+    def last_path(self):
```

Comment:
I think it would make more sense for this to be `current_path` rather than `last_path`; "last" could be interpreted as the previously active path.